dhcpv6/option_ntp_server.go

```diff
@@ -0,0 +1,157 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/insomniacslk/dhcp/rfc1035label"
+	"github.com/u-root/uio/uio"
+)
+
+// NTPSuboptionCode defines the type for NTP sub-option codes.
+type NTPSuboptionCode uint16
+
+// NTPSuboption is the interface that every NTP suboption must implement.
+type NTPSuboption interface {
```

Comment:
Do we need a new interface definition here?

I would use the same [Option interface](https://github.com/insomniacslk/dhcp/blob/d4b282fe8d4cf3f8023e01e0f17cb74c510ad228/dhcpv6/options.go#L10) we already have.

Reply:
good point. I started with a simpler interface, and then added pretty much the same methods. I'll remove it